### ISSUE TITLE
v0.4.2: Observing a build failure on Windows

### ISSUE THREAD

**Issue by lierdakil** [jgm/zip-archive#61]
```
Failed to build zip-archive-0.4.2.
cabal.exe: Failed to build zip-archive-0.4.2 (which is required by
Build log (
pandoc-2.17.1.1). See the build log above for details.
C:\cabal\logs\ghc-8.10.7\zip-archive-0.4.2-ebb[976](https://github.com/lierdakil/pandoc-crossref/runs/5394825626?check_suite_focus=true#step:11:976)8932adbdb43ef91090f80ebcf911d5c65e.log
):
Preprocessing library for zip-archive-0.4.2..
Building library for zip-archive-0.4.2..

[1 of 1] Compiling Codec.Archive.Zip ( src\Codec\Archive\Zip.hs, dist\build\Codec\Archive\Zip.o )

src\Codec\Archive\Zip.hs:492:31: error:
Error:     Variable not in scope: isLetter :: Char -> Bool
    |
492 |                (c:':':d:xs) | isLetter c
    |                               ^^^^^^^^
Error: Process completed with exit code 1.
```

https://github.com/lierdakil/pandoc-crossref/runs/5394825626

GHC 8.10.7.

**Comment by jgm** [comment]
I'll add Windows to the CI!
It's under CPP which is why it didn't come up before.

**Comment by jgm** [comment]
Should be fixed in 0.4.2.1.
